src/app/content/components/constants.ts

```diff
@@ -9,10 +9,17 @@ export const sidebarDesktopWidth = 33.5;
 export const sidebarMobileWidth = 28.8;
 export const sidebarTransitionTime = 300;
 
+export const searchResultsBarDesktopWidth = 43.5;
+export const searchResultsBarMobileWidth = 100;
+
 export const toolbarIconColor = theme.color.primary.gray;
 export const toolbarDesktopHeight = 5;
 export const toolbarMobileHeight = 4;
 
+export const mobileSearchContainerMargin = 1;
+export const toolbarSearchInputDesktopHeight = 3.2;
+export const toolbarSearchInputMobileHeight = 3.2;
```

Comment:
if we also export variables for `toolbarSearchDesktopHeight` `toolbarSearchMobileOpenHeight` `toolbarSearchMobileClosedHeight` that might make logic elsewhere simpler